doc-content/enterprise-only/openshift/template-deploy-ha-resources-proc.adoc

```diff
@@ -0,0 +1,24 @@
+[id='template-deploy-ha-resources-{context}-proc']
+= Configuring resource usage for a high-availability deployment
+
+If you are deploying the high-availability template (`{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring-ha.yaml`), you can optionally configure resource usage to optimize performance for your requirements.
```

Comment:
And more about limits and resources can be found in OpenShift documenation [https://docs.openshift.com/container-platform/3.11/dev_guide/compute_resources.html](https://docs.openshift.com/container-platform/3.11/dev_guide/compute_resources.html)

Reply:
Not required, but can be easier for users to understand using memory limits (and formats of configured values).

Reply:
To learn more about limits and resources, see (link).